scripts/variantstore/wdl/GvsPrepareBulkImport.wdl

```diff
@@ -0,0 +1,82 @@
+version 1.0
+
+import "GvsUtils.wdl" as Utils
+
+workflow GvsPrepareBulkImport {
```

Comment:
a little doc at the top to describe what this does would be nice

Reply:
also if this is a one task workflow, maybe the task could just live with the rest of the import workflow?

Reply:
I _think_ this will be more complicated when we work to make the input params less obnoxious---but it's a good point to consider